spec/fixtures/vcr_cassettes/spec/features/hearing_pagination_spec.yml

```diff
@@ -8,7 +8,7 @@ http_interactions:
       string: ''
     headers:
       User-Agent:
-      - Faraday v1.1.0
```

Comment:
Since these yml files are generated by VCR cassette to record http responses, did you [re-run the VCR cassette](https://github.com/ministryofjustice/laa-court-data-ui/blob/master/docs/testing.md#vcr-and-webmock) to get these versions updated?